### DISCUSSION TITLE
No Onscreen Keyboard

### DISCUSSION THREAD

**Discussion by kmy477** [Archfx/duo-de#106] [Q&A] [answered]
I'm new to this, just flashed my Duo1 with the latest ROM, everything seems to be working fine except there's no onscreen keyboard popped up when I tried to input text. Can some one please help? Thank you.

**Comment by wilhelmstroker** [comment, marked as answer]
Turn off pen input in gboard or use a different keyboard. Swiftkey works fine for me.
